medusa/providers/generic_provider.py

```diff
@@ -53,6 +54,7 @@
 from medusa.show.show import Show
 
 from requests.utils import add_dict_to_cookiejar
+from pytz import timezone
```

Comment:
Is there a reason to select pytz here instead of dateutil since we already use dateutil timezones?